racketlogger/ModernKiosk
# [Part 2] Improvements with EKC, password, and, notifications

The rest of file not added to last commit.

To test:
- Go to Preferences and change and save options (see all notifications).
- Populate password and save.
- Enter in Kiosk Mode.
- Press Ctrl + Shift + U to close Kiosk Mode. (check "close and show preferences")
- Test to close Kiosk Mode the Window is re-opened till EKC combination.
- Test EKC combination with and without password.

Please package version 0.13 and publish in AMO platform
